docs/GraFx-Studio/guides/layouts/index.md

```diff
@@ -6,13 +6,23 @@ Click on the **plus** sign or three dots "Add sub-layout"
 
 ![Creat first Sub-Layout](sublayout-1.gif)
 
-Rename the layout to "Skyscraper". Since your renamed, it's selected.
+A window appears to allow you to choose from a [list of Layout Presets](#reference-available-layout-preset-sizes).
+
+![screenshot-full](layoutpresets01.png)
+
+Select 1 or more presets you would like to add to your Layout List.
+
+![screenshot-full](layoutpresets02.png)
+
+If your preset is not available in the list, choose "Custom" and define the name, size and intent of your Layout.
+
+![screenshot-full](layoutpresets03.png)
 
 Select the properties panel, and change the dimentions to 160 x 600.
```

Comment:
This could be confusing because above you say that with the layout presets or custom mode, you can set the size when creating a layout. Then you say that you change the dimensions in the properties panel, but this step is optional with the latest changes.